test/controllers/UploadControllerSpec.scala

```diff
@@ -433,9 +448,10 @@ class UploadControllerSpec extends FrontEndTestHelper {
     "throw an error for invalid input data" in {
       val graphQLConfiguration: GraphQLConfiguration = new GraphQLConfiguration(app.configuration)
       val consignmentService: ConsignmentService = new ConsignmentService(graphQLConfiguration)
+      val fileStatusService = new FileStatusService(new GraphQLConfiguration(app.configuration))
```

Comment:
This is the same as the comment below but you can reuse the `graphQLConfiguration` variable here. I know you didn't add it in but if you could also update the ones for the `uploadService` variable as we're calling `new GraphqlConfiguration` there as well. It keeps things a bit tidier.

Reply:
Updated for `uploadService` too.